### PR TITLE
fix: added extracting in_player_env from configuration instead storage

### PR DESCRIPTION
**Pull request by sm00th1980**
## Description

[MYTVTOGO & Kanal - Samsung - Content is not clickable](https://applicaster.monday.com/boards/1615228456/pulses/2497183506)

`For inplayer_login plugin added retrieve in_player_enviroment value from configuration instead storage`

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [x] Samsung TV (Tizen)
- [ ] Web
</details>

### CODE REVIEW

File: plugins/quick-brick-inplayer-login/runInBackground.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
+import * as R from "ramda";
+
 import { getLoginProtocol } from "./src/Services/loginPluginProtocol";
 
 export const runInBackground = async (item, callback, configuration, presentUI) => {
-  const { isLoggedIn } = await getLoginProtocol();
+  const in_player_environment = R.pathOr(undefined, ['configuration', 'in_player_environment'], configuration)
```
Comment (Budaa) [inline plugins/quick-brick-inplayer-login/runInBackground.js:6]: R.path should be enough in this case.
```suggestion
  const in_player_environment = R.path(['configuration', 'in_player_environment'], configuration)
```

**Review by Budaa — APPROVED** [review]

**Comment by inckie** [comment]
Login protocol should not receive this argument, its an interface used across multiple plugins so they can be called from the outside. This change will break multilogin plugin.
Instead we need a way for plugins to retrieve configurations from any place inside, since its does not change after start, and should not be dependent from any react code.

**Comment by sm00th1980** [comment]
> Login protocol should not receive this argument, its an interface used across multiple plugins so they can be called from the outside. This change will break multilogin plugin. Instead we need a way for plugins to retrieve configurations from any place inside, since its does not change after start, and should not be dependent from any react code.

@inckie Thanks for clarification. I'll revert this PR.

**Comment by sm00th1980** [comment]
> @sm00th1980 lets look for better way to get this data, I remember we took it from storage since we did not find a better way without violating API. From the other hand soon there will be new API in QB for plugin config that Avi team is doing, so maybe it will be easier.

@inckie sorry, I don't understand what options do we have for passing this data?

**Pull request merged**